GitCommands/ArgumentBuilderExtensions.cs

```diff
@@ -226,6 +226,10 @@ string GetArgument()
                         return "--mixed";
                     case ResetMode.Hard:
                         return "--hard";
+                    case ResetMode.Merge:
```

Comment:
I suggest the selected order is the same in the code as in the GUI - dont change this yet though, I am not sure of the order myself...